dev-docs/documentation/contribute_docs.md

```diff
@@ -61,6 +61,18 @@ This will build *and serve* the documentation at http://127.0.0.1:8000 -- this
 gives you a live preview of any changes you make (and save) to the
 documentation without needing to continually rebuild.
 
+## Doc testing
+
+Before running the `make` command, make sure to download the python venv module by running the following command:
+```sh
+apt install python3-venv
+```
+When working with a `docs` branch and making updates, you can run the documentation tests using the Makefile. 
```

Comment:
I think this applies to both `docs` and `docs-devel`. I'd prefer us to be specific here, because the implication of "a docs branch" is "any" docs branch. Alternatively, I guess we could say "with our docs-related branches". Either would be clearer, so happy to leave the choice up to you :)